Jessy/email_ids_challenge.py

```diff
@@ -17,11 +18,11 @@
 # 1.3 TODO: We want to join the first name and last name with a '.' and save it in a variable called 'joined_names'
 # joined_names
 # This was tough, had to look for videos online. I don't remember viewing .split, .join videos on Canvas.....But I love it1!!
-joined_names='.'.join(name_list)
-print(joined_names)
+joined_name='.'.join(name_list)
+print(joined_name)
 # 1.4 TODO: We want to add '@ripplemedia.com' to the end of the string inside joined_names and save it in a variable 'email'
 # I had a hard time with this question
-email=joined_names+"@ripplemedia.com"
+email=joined_name+"@ripplemedia.com"
```

Comment:
I haven't looked into this, but it would be great if VS code allowed you to "Find all and Replace with" functions like excel